crates/tables/src/lib.rs

```diff
@@ -68,7 +68,7 @@ tables!(
         val md5: String,
     }
 
-    table DataPlanes (row DataPlane, sql "data_planes") {
+    table DataPlanes (row #[derive(Clone, PartialEq)] DataPlane, sql "data_planes") {
```

Comment:
why is PartialEq required? That's a bit odd to me.

Reply:
it wasn't. Removed